localstack/config.py

```diff
@@ -245,11 +245,28 @@ def ping(host):
 
 
 def in_docker():
-    """ Returns True if running in a docker container, else False """
+    """
+    Returns True if running in a docker container, else False
+    Ref. https://docs.docker.com/config/containers/runmetrics/#control-groups
+    """
     if not os.path.exists('/proc/1/cgroup'):
         return False
+    try:
+        if any([
+            os.path.exists('/sys/fs/cgroup/memory/docker/'),
+            any(['docker-' in file_names for file_names in os.listdir('/sys/fs/cgroup/memory/system.slice')]),
+            os.path.exists('/sys/fs/cgroup/docker/'),
+            any(['docker-' in file_names for file_names in os.listdir('/sys/fs/cgroup/system.slice/')]),
+        ]):
+            return False
+    except FileNotFoundError:
```

Comment:
Looks like the CI build are failing with:
```
module initialization error: global name 'FileNotFoundError' is not defined
```
I guess we can simplify to:
```
     except Exception:
```
?

Reply:
I see, thanks.